deploy.sh

```diff
@@ -23,7 +23,7 @@ fi
 
 msg "Removing the old website"
 pushd public
-git rm -rf *
+git rm -rf public/*
```

Comment:
Of course, I ignored `pushd` by mistake. And I agree that the single line `rm -rf public/` simplifies things which is good if you can't remember why not earlier, although keeping `git` in front saves another `git add` line, so `git rm -rf public/` in the end, correct?